### ISSUE TITLE
Uses serif font on Manjaro

### ISSUE THREAD

**Issue by fonorobert** [KELiON/cerebro#155]
Running Manjaro (Arch based), installed cerebro from AUR from `cerebro-git`.
cerebro works correctly in every other regard but it seems to default to a serif font instead of either the sans font seen in screenshots or the system default font (also sans). see screenshot:

![screenshot from 2017-02-25 11-35-50](https://cloud.githubusercontent.com/assets/6963710/23330534/65890050-fb50-11e6-9855-04990b336d47.png)

I'd be happy to help investigate if someone can give me pointers on where to start.

**Comment by maximbaz** [comment]
I may be wrong, but I think [this](https://github.com/jonathantneal/system-font-css) is what is used to determine the font.

By the way, consider trying out [fonts-meta-extended-lt](https://aur.archlinux.org/packages/fonts-meta-extended-lt/), I'm very happy with these fonts.

![image](https://cloud.githubusercontent.com/assets/1177900/23334729/738a0960-fba5-11e6-83c3-b8f45d54c844.png)

**Comment by dar5hak** [comment]
Related: On elementary OS, everything uses the Ubuntu font, except Cerebro Settings, which uses serif.

![screenshot from 2017-02-26 23 05 38](https://cloud.githubusercontent.com/assets/8205055/23342000/2e984e38-fc78-11e6-89c3-9285bea06ad4.png)

**Comment by Razzile** [comment]
I think the cause of this may be this line https://github.com/KELiON/cerebro/blob/d994e0b62ecc5d8af17e42efbac48642098ce047/app/main/plugins/core/cerebro/settings/Settings/styles.css#L6

Helvetica Neue isn't available on anything except macOS afaik (without downloading it separately)

**Comment by kuzzmi** [comment]
@dar5hak Using Manjaro i3 edition (I guess it uses the same font family as in official distros) and the default font is [Noto Sans](https://fonts.google.com/specimen/Noto+Sans)

**Comment by fonorobert** [comment]
Sorry @dar5hak for not responding. In the gnome and cinnamon editions the default font is Cantarell.

**Comment by jalopezsuarez** [comment]
I have the same problem in Debian8. The solution is put:
`>> dark.css
:root {
  /* Main fonts and colors */
  --main-background-color: rgba(62, 65, 67, 1);
  --main-font: sans-serif;
  --main-font-color: white;
`
sans-serif instead system to get Arial style font, because system get a serif font like Time News Roman.

**Comment by maximbaz** [comment]
Since you are unhappy with the font that system chooses for `sans-serif`, have you guys considered adjusting your system configuration for default font? It will make _all_ your apps nicer.

If there is no UI to do this, it's a matter of a relatively simple fontconfig configuration:

```
<?xml version='1.0'?>
<!DOCTYPE fontconfig SYSTEM 'fonts.dtd'>
<fontconfig>
  <alias>
    <family>serif</family>
    <prefer><family>Heuristica</family></prefer>
  </alias>
  <alias>
    <family>sans-serif</family>
    <prefer><family>Noto Sans</family></prefer>
  </alias>
</fontconfig>
```

Put this in `~/.config/fontconfig/conf.d/30-better-fonts.conf` and run `$ fc-cache -f; sudo fc-cache -f`.

**Comment by jalopezsuarez** [comment]
Ok... thanks for the solution. I was working around your proposal and found a solution for Debian8.

I fond the way to check the problem, we need to check what font is associated with `system` as you put in the CSS in Cerebro app.

Next command will give you information what fonts is associated with system family:
`fc-match --verbose system`

Then create or edit your user fontconfig configuration file `~/.config/fontconfig/fonts.conf`, and add system alias:

`<?xml version='1.0'?>
<!DOCTYPE fontconfig SYSTEM 'fonts.dtd'>
<fontconfig>
  <alias>
    <family>system</family>
    <prefer><family>DejaVu Sans</family></prefer>
  </alias>
</fontconfig>`

After that reload default system fonts:
fc-cache -f; sudo fc-cache -f
` 

Solved! Thanks

**Comment by maximbaz** [comment]
Yeah, actually you are right, you might have to define the alias for `system` font too. I'll adjust my snippet above.
